app/controllers/static_pages_controller.rb

```diff
@@ -0,0 +1,7 @@
+class StaticPagesController < ApplicationController
+  def home
+  end
```

Comment:
```suggestion
  def home; end
```
ri nhé, coding convention là nếu phân fthaan method ko có thì viết dồn lên 1 line

Reply:
ready